guides/common/attributes-base.adoc

```diff
@@ -29,6 +29,44 @@
 :RepoRHEL7ServerSatelliteServerPuppetVersion: rhel-7-server-satellite-6.3-puppet4-rpms
 :RepoRHEL7ServerSatelliteCapsulePuppetVersion: rhel-7-server-satellite-capsule-6.3-puppet4-rpms
 
+//Added by Marie for GA
```

Comment:
this block does clash with what we have in lines 16 to 27 -- should it replace it instead?

Reply:
Hi @evgeni , you're right. I meant to remove the old attributes used for 6.10 later. I agree it looks confusing so I've gone ahead and removed them. Thank you!